spyderlib/widgets/ipython.py

```diff
@@ -293,6 +293,8 @@ def get_signature(self, content):
         data = content.get('data', {})
         text = data.get('text/plain', '')
         if text:
+            if type(text) == type({'e':5}):

```

Comment:
Well, it's just a temporary patch at the wrong place, so that I can ship 3.0devb3+
I've no idea of the true/nice patch